sounds/templatetags/display_sound.py

```diff
@@ -148,35 +149,35 @@ def display_sound_small_with_timesince(context, sound):
-    return display_sound(context, sound, player_size='moderation', show_bookmark=False, show_similar_sounds=False, show_remix=True, show_rate_widget=False)
+    return display_sound(context, sound, player_size='moderation', show_collections=False, show_bookmark=False, show_similar_sounds=False, show_remix=True, show_rate_widget=False)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_small_no_bookmark(context, sound):
-    return display_sound(context, sound, player_size='small', show_bookmark=False, show_similar_sounds=False, show_remix=False, show_rate_widget=True)
+    return display_sound(context, sound, player_size='small', show_collections=False, show_bookmark=False, show_similar_sounds=False, show_remix=False, show_rate_widget=True)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_small_no_bookmark_no_ratings(context, sound):
-    return display_sound(context, sound, player_size='small', show_bookmark=False, show_similar_sounds=False, show_remix=False, show_rate_widget=False)
+    return display_sound(context, sound, player_size='small', show_collections=False, show_bookmark=False, show_similar_sounds=False, show_remix=False, show_rate_widget=False)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_middle(context, sound):
-    return display_sound(context, sound, player_size='middle', show_bookmark=True, show_similar_sounds=True, show_remix=True)
+    return display_sound(context, sound, player_size='middle', show_collections=True, show_bookmark=True, show_similar_sounds=True, show_remix=True)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_big_no_info(context, sound):
     return display_sound(context, sound, player_size='big_no_info')
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_big_no_info_no_bookmark(context, sound):
-    return display_sound(context, sound, player_size='big_no_info', show_bookmark=False, show_similar_sounds=False, show_remix=False)
+    return display_sound(context, sound, player_size='big_no_info', show_collections=False, show_bookmark=False, show_similar_sounds=False, show_remix=False)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_small_no_info(context, sound):
     return display_sound(context, sound, player_size='small_no_info', show_rate_widget=True)
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
 def display_sound_small_no_info_no_buttons(context, sound):
-    return display_sound(context, sound, player_size='small_no_info', show_rate_widget=False, show_bookmark=False, show_similar_sounds=False, show_remix=False)
+    return display_sound(context, sound, player_size='small_no_info', show_rate_widget=False, show_collections=False, show_bookmark=False, show_similar_sounds=False, show_remix=False)
```

Comment:
its not directly related to this PR, but I see that `show_rate_widget` comes _after_ the bookmark/collections params. It'd be good for these named parameters to be in the same order as they are defined in the method